plugin/bookmark.vim

```diff
@@ -253,7 +253,9 @@ function! CallDeprecatedCommand(fun, args)
   return call(Fn, a:args)
 endfunction
 
-function! BookmarkModify(diff)
+function! BookmarkModify(diff, ...)
```

Comment:
You mean add a source comment, or get rid of `...` and use a named argument instead?
`...` is there so that you can pass a boolean optional argument (without affecting existing call sites of this function) which, when `true` (or `!= 0`), will treat `diff` as an absolute line number instead of a relative offset. This is so you can run the command `:BookmarkMoveToLine 100` and the current bookmark is moved to line 100 (instead of having to move it N times). I felt the need for such a command when I misplaced an annotation, which I would either have to move ~30 lines up, or delete and recreate/retype it at the correct line.

The core logic of the function is essentially the same.